src/main/resources/fabric.mod.json

```diff
@@ -70,8 +70,11 @@
   ],
   "depends": {
     "fabricloader": ">=0.12.0",
-    "fabric-rendering-data-attachment-v1": ">=0.1",
-    "fabric-rendering-fluids-v1": ">=0.1"
+    "fabric-block-view-api-v2": "*",
```

Comment:
There was a mistake and originally I felt it was inaccurate, so I will double check what minimum version of each module Sodium actually needs and add those instead of the wildcard.

Reply:
After double checking, the parts of `fabric-rendering-data-attachment-v1` that Sodium uses/expects are present in every version of it. The parts of `fabric-rendering-fluids-v1` that Sodium uses/expect only exist in version 2.0.0 and onward. All other dependencies are also correct.